index.html

```diff
@@ -12,20 +12,26 @@
 <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;400;500;700;800&family=Poppins:wght@400;700&family=Roboto:wght@300;400;700&display=swap" rel="stylesheet">
 </head>
 <body>
+  <div id="menu-section" class="menu-section">
+          <button id="menu-toggle" class="menu-toggle"><i class="fa-solid fa-xmark"></i></button>
+        <ul class="menu-list">
+          <li class="menu-item"><a href="#portfolio">Portfolio</a></li>
+          <li class="menu-item"><a href="#about">About</a></li>
+          <li class="menu-item"><a href="#contact">Contact</a></li>
```

Comment:
- [ ] when I open the mobile menu and click a link to a section, there's a bug preventing me from scrolling or clicking anywhere, kindly fix that 🙏 


![bug](https://user-images.githubusercontent.com/89282221/217259971-98d51b8b-ab67-40ee-8528-949278dd7b12.gif)